templates/bastion-inventory.j2

```diff
@@ -0,0 +1,2 @@
+[bastion]
+bastion ansible_ssh_host={{ hostvars['bastion']['ansible_ssh_host'] }}
```

Comment:
I was thinking about adding that, but my thought was to later include user creation into the configuration step, so then setting `ansible_ssh_user=cloud-user` might interfere with later post-deployment configuration. Not all users who manage an existing bastion host necessarily have or need to have access to the cloud-user user.